README.md

```diff
@@ -77,9 +77,9 @@ Apart from MMDetection, we also released [MMEngine](https://github.com/open-mmla
 
 **v3.0.0rc3** was released in 4/11/2022:
 
-- Support training detection models in Detectron2
```

Comment:
please remember to merge the latest `dev-3.x` branch

Reply:
Done